tests/AutoReview/FixerFactoryTest.php

```diff
@@ -312,9 +311,18 @@ public function testFixerWithNoneDefaultPriorityIsTested(): void
         foreach ($fixers as $fixer) {
             $fixerName = $fixer->getName();
 
-            if (0 !== $fixer->getPriority() && !isset($fixerNamesWithTests[$fixerName])) {
-                $missing[$fixerName] = true;
+            // if method "getPriority" is NOT defined in the fixer itself then skip it
```

Comment:
why?
what if there is
- AbstractFixer
- PhpDocAbstractFixer // defining priority
- PhpDocReturnFixer // not defining priority

i would still expect to test priority declared in PhpDocAbstractFixer

Reply:
> what if there is
> AbstractFixer

Like the one that every fixer is extending? This is the one in fact providing the "default" priority.

> i would still expect to test priority declared in PhpDocAbstractFixer

How would you want to have it? Keep in mind that this test is based on a priority graph, so you would have to... actually I have no idea what magic would be required.

Take a look what other changes it forced - is it improvement or not?

Reply:
```
AbstractFixer -> PhpDocAbstractFixer -> PhpDocReturnFixer
```

vs

`if method "getPriority" is NOT defined in the fixer itself then skip it`

that's what I challenge. `getPriority` defined in `AbstractFixer` could be ignored, but one from `PhpDocAbstractFixer` likely shouldn't

Reply:
All the `Abstract*Fixer` are not defining `getPriority` method, but I get your point. I've updated it so only `AbstractFixer` and `AbstractProxyFixer` make are the exceptions.

Reply:
why `AbstractProxyFixer` ?
if `FooFixer` proxies to `BarFixer`, then both of them should have priority checked in same way

Reply:
> both of them should have priority checked in same way

what would be the point of having the same test twice?

Reply:
how do you know it works otherwise?
sometimes proxy fixers are tricky, reconfigure things, put other config, or put sth on top of proxied fixer. let's not assume, let's ensure by test.

Reply:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7083#event-9831594802

I'm still missing how to continue with review, due to question above.